### ISSUE TITLE
Error when loading eye attributes via character tab (Before patch 6.1)

### ISSUE THREAD

**Issue by ShiftyKiwi** [imchillin/Anamnesis#943]
When loading eye attributes for either masculine or feminine miqote models, the first option or eyes 1, is not able to be loaded in by: default when first opening the program, manually selecting it, or via loading in a saved miqote appearance; Instead, the second option or eyes 2 is the default that it will load in as. Similar testing was done to try and replicate the issue on other playable races, but seemingly they are unaffected by this error when using the above methods of loading eyes 1.

1. Open anamnesis.
2. Select eyes 1 option in the character looks tab when already playing a masculine or feminine miqote model, or trying to load an saved miqote appearance in that whose eyes are eyes 1.
3. Reload the characters to still be stuck on eyes 2.

**Comment by ShiftyKiwi** [comment]
The other races/tribes that I had tested this on were midlander, and roes. Unfortunately at the time, I didn't think to take into account both of these races respective tribes. When this issue first occurred, it had been on Sunseekers.

**Comment by Ani-ki** [comment]
No worries. Once we merge Asgard's PR in I can take a look at this as well.

**Comment by AsgardXIV** [comment]
I think this is similar to #923. 
It seems like you can't set eyes to the minimum value of the slider for some reason.

**Comment by ShiftyKiwi** [comment]
I went ahead and did further testing to try and replicate this issue on all other races, and tribes. And setting the values back to the minimum/eyes 1 did not change the initial outcome. 

At first I had been confused as to how I was able to get the minimum value/eyes 1 setting loaded when I tried it on Midlander and Roe, but I remembered that when I had tried the method of loading in an appearance, I had managed to get the minimum values/eyes 1 via two ways:

1. Loading in a saved character appearance file from the character creation method, whom already had the minimum values/eyes 1 setting chosen.
2. Loading in an NPC appearance (whom coincidentally, also had the same minimum/eyes 1 values). In the case of loading in to attain minimum values with the NPC appearance, I had chosen 'Warrior of Darkness.' and for the Roe, 'Biggs.' . 

While these methods had been able to successfully load the minimum values/eyes 1, I also noticed another issue. If at any point I changed these minimum/eyes 1 values to something beyond it, I could not change them back, and I would have to go about the two methods above, to get that same option again.

I hope that this information can shed more light, and I apologize if my writing is a verbose. Thank you for taking the time and consideration to help me!
